templates/publisher/measure.html

```diff
@@ -120,7 +119,10 @@ <h3 class="u-float--left">Retention</h3>
   snapcraft.publisher.selector('.metrics-period', 'period');
   snapcraft.publisher.selector('.active-devices', 'active-devices');
   snapcraft.publisher.metrics({
-    activeDevices: {{ active_devices|safe }},
+    activeDevices: {
+      metrics: {{ active_devices|safe }},
```

Comment:
I think there is a `json` or (`tojson`?) filter that may be better to use here instead of `safe`. 
Unless it's HTML that is rendered here.

Reply:
I tried tojson and it didn't seem to work - flask route converts to json before sending it to the template, which I guess is the reason?

Reply:
Ok.